### PR TITLE
Allow wait for request completion in command

### PR DESCRIPTION
**Pull request by schaefi**
Added --wait for async request, update and fixup the manual pages along with the change

Related to: https://trello.com/c/TDNWTWZW/167-azurectl-next-1

### CODE REVIEW

**Comment by schaefi** [comment]
I want to have that in and as follow up the wait for an instance to be ready for access before the priority tasks e.g VM reboot. That makes things a little bit easier.

**Comment by schaefi** [comment]
no objections and a pretty simple change set let me think this can be merged in order to unblock the prio task implementations

File: azurectl/storage/account.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ def update(
             raise AzureStorageAccountUpdateError(
                 '%s: %s' % (type(e).__name__, format(e))
             )
-        return self.show(name)
+        return result.request_id
```
Comment (bear454) [inline azurectl/storage/account.py]: This is an unexpected change in the behavior of the API.

File: doc/man/azurectl::compute::data_disk.md [hunk 2]
```diff
@@ -26,19 +27,17 @@ __azurectl__ compute data-disk show --cloud-service-name=*name* --lun=*lun*
...
+    [--wait]
 
 # DESCRIPTION
 
 ## __create__
 
-Create a new empty disk, and attach it to the selected virtual machine. If the
-virtual machine's __instance-name__ is the same as the __cloud-service-name__,
-the __--instance-name__ argument may be omitted.
+Create a new empty disk, and attach it to the selected virtual machine. If the virtual machine's __instance-name__ is the same as the __cloud-service-name__, the __--instance-name__ argument may be omitted.
```
Comment (bear454) [inline doc/man/azurectl::compute::data_disk.md]: I don't consider line breaks at 80 chars to be a "markup issue".

**Comment by bear454** [comment]
There are a couple of issues in this PR that were at least worth discussion, including a change to user output, and dropping the 80-char margin in docs.

Considering that we were all offline Friday, and that your two main reviewers would be travelling over the weekend, you should have waited for a review before merging. Regardless of conditions, we have an agreement to wait 7 days.

File: azurectl/storage/account.py (hunk 1 above)
Comment (schaefi) [inline azurectl/storage/account.py]: > > -        return self.show(name)
> > +        return result.request_id
> 
>    This is an unexpected change in the behavior of the API.

The return value was never used in the code, and all async methods
returns the request_id except for this one, which I consider a bug
we overlooked

File: doc/man/azurectl::compute::data_disk.md (hunk 2 above)
Comment (schaefi) [inline doc/man/azurectl::compute::data_disk.md]: >    I don't consider line breaks at 80 chars to be a "markup issue".

yes they are, I'm sorry the rendering of the man pages only works correctly if we don't break the lines ourselves

**Pull request merged**